crates/runtime/tests/models/openai.rs

```diff
@@ -178,7 +212,7 @@ async fn openai_test_search() -> Result<(), anyhow::Error> {
     tracing::info!("/v1/search: Ensure simple search request succeeds");
     let response = send_search_request(
         http_base_url.as_str(),
-        "vehicles and journalists",
+        "new patient",
```

Comment:
Replaced previous search term as there are two identical records for that term in the dataset and their ordering was flaky due to scoring is the same.